### ISSUE TITLE
npm test fails

### ISSUE THREAD

**Issue by johngriffin** [truffle-box/drizzle-box#10]
Hi, great library!  Unfortunately npm test is failing for me, running on Mac OS X High Sierra.  

I've tried removing node_modules and doing an npm install.  No dice.  Looks like the path is wrong in package.json.

```
FAIL  src/App.test.js
  ● Test suite failed to run

    ENOENT: no such file or directory, stat '/Users/john/dev/drizzle/\config\polyfills.js'
        at Error (native)
      
      at Object.fs.statSync (fs.js:981:18)
      at Object.statSync (node_modules/graceful-fs/polyfills.js:297:22)

Test Suites: 1 failed, 1 total
```

**Comment by drjiro** [comment]
HI, I also try it on Mac. If you try it on not Windows,  It may occur for Win32 file system, so that change from "\\" to "/".  Modify the following lines in 'package.json'.

line 76
      "<rootDir>/config/polyfills.js" 

line 85-86
      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
      "^(?!.*\\.(js|jsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"

I will pass the test. However, I cannot run it on Chrome by another problem.

**Comment by pemulis** [comment]
I have the same error as @johngriffin, and got past it by changing just line 76 of package.json:

`"<rootDir>/config/polyfills.js"`

But now I have a new error:

    FAIL  src/App.test.js
    ● Test suite failed to run

    Your test suite must contain at least one test.
      
      at onResult (node_modules/jest/node_modules/jest-cli/build/TestRunner.js:189:18)
          at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:160:7)

Working on debugging that!

**Comment by pemulis** [comment]
Part of the problem might be that I still had my local test server running while trying to reinstall my node modules. (I had run `npm run start` in a different Terminal tab.) So I rebooted that, deleted my node_modules/ directory, and reinstalled.

Getting this error now, which leads me to believe I need to change more lines in package.json, as drjiro suggested:

       FAIL  src/App.test.js
        ● Test suite failed to run

    Cannot find module '/Users/j/Coding/cryptokittybox/\config\jest\cssTransform.js'
      
      at Function.Module._resolveFilename (module.js:555:15)
      at Function.Module._load (module.js:482:25)
      at Module.require (module.js:604:17)
      at require (internal/module.js:11:18)

      Test Suites: 1 failed, 1 total
      Tests:       0 total
      Snapshots:   0 total
      Time:        1.673s
      Ran all test suites.

**Comment by pemulis** [comment]
Here's what I changed in package.json to get the test to pass:

Line 76 
`"<rootDir>/config/polyfills.js"`

Lines 85-86
`      "^.+/.css$": "<rootDir>/config/jest/cssTransform.js",`

`      "^(?!.*/.(js|jsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"`

I was still getting a "Cannot find module" error until I added <rootDir> to those lines.

**Comment by am17torres** [comment]
I'm on OS X Sierra using Visual Studio Code 1.21.1

@pemulis, are you sure your snippet is correct?
Shouldn't `"^.+/.css$"` actually be `"^.+\\.css$"`
And the same for `"^(?!.*/.(js|jsx|css|json)$)":` -> `"^(?!.*\\.(js|jsx|css|json)$)":`

Doing that got rid of the error `Your test suite must contain at least one test.` In _src/App.test.js_ and _scripts/test.js_. 

*Updates:*
* Bumped `jest` and `babel-jest` to 22.4.1 because the VS Code [jest extension](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest) was complaining.
Now I'm getting `SyntaxError: Identifier 'jest' has already been declared`. Not sure if that is the result of bumping the version which is probably unrelated to this issue.

![screen shot 2018-03-26 at 12 25 09 am](https://user-images.githubusercontent.com/70786/37886879-30204802-308c-11e8-99e4-0f5ac50ef5a2.png)

**Comment by JakeGlascock** [comment]
Had a clean unbox of drizzle.  I get 'ReferenceError: [BABEL] /apps/drizzle01202019/app/src/App.test.js: Unknown option: base.configFile. Check out http://babeljs.io/docs/usage/options/ for more information about options.' at the 
`npm run test`
step.

Using nvm to use v11.7.0 of node, npm v6.5.0.  

After much research (doge style), I found a solution.

`truffle unbox` must use npm and not yarn as https://github.com/facebook/create-react-app/issues/5259 was the error I was getting.  This solution suggested using yarn as 

> as it's reportedly correctly hoisting packages

So after unboxing, I deleted the app/node_modules directory and re-installed from the app directory with 

`yarn install`

Yarn v1.13.0

After that, the out of the box test that comes with the drizzle box passed using 

`yarn run test`
